transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -158,6 +162,9 @@ public int sendToAddress(long memoryAddress, int pos, int limit, InetAddress add
         if (res >= 0) {
             return res;
         }
+        if (res == Errors.ERROR_ECONNREFUSED_NEGATIVE) {
+            throw new PortUnreachableException("sendto failed");
```

Comment:
May be "sendToAddress failed" will more informative?
And similar at sendToAddresses().